player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -324,7 +325,11 @@ private void setEmptyState() {
         mQuartile = 0;
         mTrackingEventMap = null;
         mProgressTracker = null;
-        mBannerBitmap = null;
+        if (mIsBannerSet) {
```

Comment:
if leave this field without nullifying it, it's possible that for next ad it displays the banner of previous one.